src/lib/coffee-recommendation-algorithm.js

```diff
@@ -0,0 +1,43 @@
+import coffees from "../data/coffees"
+import moods from "../data/moods"
+
+// moods : a list of mood indexs user have selected
+// temperature : decimal temp valu fetched from OpenWeatherAPI
+function getCoffee (selectedMoods, temperature) {
+    let result = ""
+    if(temperature < 10) {
+        result += "hot "  
+    } else if (temperature >= 17) {
+        result += "iced "
+    }
+
+    // Calculate the total score
+    let total = [0,0,0,0,0]
```

Comment:
Could we dynamically initialize `total` and rename it to `totalScoresArray`? 
```suggestion
    const totalScoresArray = new Array(coffees.length).fill(0)
```